themes/base.theme.bash

```diff
@@ -74,7 +74,7 @@ THEME_SHOW_USER_HOST=${THEME_SHOW_USER_HOST:=false}
 USER_HOST_THEME_PROMPT_PREFIX=''
 USER_HOST_THEME_PROMPT_SUFFIX=''
 
-VIRTUAL_ENV=
+#VIRTUAL_ENV=
```

Comment:
Instead of commenting out, can you please remove this line (and the blank line after that)?

Reply:
Doh! Of course!